mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -1081,8 +1081,7 @@ static void populateDefaults() {
         paddingWidthLeft.getValue(), paddingWidthRight.getValue(),
         strideWidth.getValue(), dilationWidth.getValue());
   }
-  if (isConv && outputDepth.getNumOccurrences() == 0 &&
-      inputDepth.getNumOccurrences() > 0) {
+  if (isConv && outputDepth.getNumOccurrences() == 0) {
```

Comment:
If we don't specify an outputDepth, we should make one, regardless of whether we specified an inputDepth.  The latter can safely default and we can work with that.